tools/build/cross-build/include/linux/fts.h

```diff
@@ -0,0 +1,10 @@
+
+#pragma once
+
+#include_next<fts.h>
+
+#ifdef __GLIBC__
```

Comment:
Doing this in the wrapper is a good idea, I patched the source in my upstream changes...